template-for-repository/proofs/Makefile.common

```diff
@@ -395,6 +412,8 @@ clean:
 	-$(RM) $(DEPENDENT_GOTOS)
 	-$(RM) TAGS*
 	-$(RM) *~ \#*
+	-$(RM) Makefile.arpa
+	-$(RM) -r $(ARPA_BLDDIR)
```

Comment:
I wonder whether these two should actually be in `veryclean`. @markrtuttle ?

Reply:
I advocate putting into veryclean only those things that take a long time to rebuild.  This is a left over from the old days.  But cbmc results and the html report are the kinds of things I would put into veryclean.